### DISCUSSION TITLE
vcpkg integration?

### DISCUSSION THREAD

**Discussion by jezzzzza** [ihedvall/mdflib#60] [Ideas]
is it planned to add mdflib (maybe also utillib) to vcpkg any time in the future?

I made a little vcpkg integration on my hard drive.. maybe you can take it from there..

[vcpkg-registry.zip](https://github.com/ihedvall/mdflib/files/13501306/vcpkg-registry.zip)

**Comment by Simplxss** [comment]
A little difficult....A lot of things need to be done.

**Comment by ihedvall** [comment]
OK. I assume you want to include mdflib and utilib to the vcpkg environment. I think this is in theory possible but currently there is not enough people involved. There is no organization that backup what I'm doing. I work on free time. The mdflib and utillib are the smaller support libraries. The a2llib and odslib are monsters (godzilla libraries!?).

**Comment by ihedvall** [comment]
There is a discussion tab alternative a project tab in GitHub that might be better for this types of threads.
